suseconnect-keepalive.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Run SUSEConnect --keepalive
+Wants=suseconnect-keepalive.timer
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/SUSEConnect --keepalive
+
+# Filter out this special exit code, since it will be raised by `SUSEConnect
+# --keepalive` whenever a system is not registered, which is a pretty typical
+# scenario.
+SuccessExitStatus=71
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Is this what is wanted, that it should also be started every boot?

Reply:
this is copied from the original version: https://github.com/SUSE/connect/blob/master/package/suseconnect-keepalive.service#L15

Reply:
I know and it is not a blocker, but that doesn't answer the question. Is it also a bug in the original version?

Reply:
Then lets wait for an answer from the authors of original.

Reply:
That looks like an unwanted side effect. The plan was only to run the service by the timer.

Reply:
Then we can remove this line and the one above.